packages/node_modules/@ciscospark/react-component-activity-list/src/index.js

```diff
@@ -29,7 +29,7 @@ export default function ActivityList(props) {
       );
     }
     case ITEM_TYPE_ACTIVITY: {
-      const {activity, avatarUrl, isPending, isAdditional, isFlagged, isSelf} = visibleActivity;
+      const {activity, avatarUrl, isError, isPending, isAdditional, isFlagged, isSelf} = visibleActivity;
```

Comment:
This is the the only case that needs these values and it is from the `visibleActivity` object that is only available after the `.map`.